packages/elastic-datemath/package.json

```diff
@@ -4,5 +4,8 @@
   "description": "elasticsearch datemath parser, used in kibana",
   "license": "Apache-2.0",
   "main": "index.js",
-  "typings": "index.d.ts"
+  "typings": "index.d.ts",
+  "peerDependencies": {
+    "moment": "^2.24.0"
```

Comment:
The peer dependencies needs to be defined?

Reply:
It doesn't but has this package is still published on the npm, I thought it was better to add it here